### PR TITLE
HPCC-28461 Add listen queue limit to dafilesrv

### PR DESCRIPTION
**Pull request by mckellyln**
## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [x] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [x] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

File: fs/dafsserver/dafsserver.cpp [hunk 1]
```diff
@@ -3629,7 +3629,7 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
     IMPLEMENT_IINTERFACE
 
     CRemoteFileServer(unsigned maxThreads, unsigned maxThreadsDelayMs, unsigned maxAsyncCopy, IPropertyTree *_keyPairInfo)
-        : asyncCommandManager(maxAsyncCopy), stdCmdThrottler("stdCmdThrotlter"), slowCmdThrottler("slowCmdThrotlter"), keyPairInfo(_keyPairInfo)
+        : asyncCommandManager(maxAsyncCopy), stdCmdThrottler("stdCmdThrottler"), slowCmdThrottler("slowCmdThrottler"), keyPairInfo(_keyPairInfo)
```
Comment (mckellyln) [inline fs/dafsserver/dafsserver.cpp:3632]: typo fix while here

File: fs/dafsserver/dafsserver.hpp [hunk 2]
```diff
@@ -31,22 +31,6 @@
 #define DAFILESRV_STREAMREAD_MINVERSION 22
 #define DAFILESRV_STREAMGENERAL_MINVERSION 25
 
-// RemoteFileServer throttling defaults
```
Comment (mckellyln) [inline fs/dafsserver/dafsserver.hpp:34]: These are already defined dafscommon.hpp include

File: system/jlib/jsocket.hpp [hunk 3]
```diff
@@ -34,7 +34,11 @@
 # define _TRACELINKCLOSED
 #endif
 
+#ifdef _WIN32
 #define DEFAULT_LISTEN_QUEUE_SIZE    200            // maximum for windows 2000 server
+#else
+#define DEFAULT_LISTEN_QUEUE_SIZE    600
```
Comment (mckellyln) [inline system/jlib/jsocket.hpp:40]: I think a larger default makes sense while here

**Comment by mckellyln** [comment]
Test Build / build-gh_runner-windows-2022 / build-gh_runner (pull_request) failure
Seems to be related to vcpkg icu build on Windows and not related to this PR ?

**Review by jakesmith — APPROVED** [review]
@mckellyln - looks good to me.

**Comment by mckellyln** [comment]
ok to merge

**Pull request merged**